consensus-types/payload-attribute/getters.go

```diff
@@ -61,8 +61,8 @@ func (a *data) PbV2() (*enginev1.PayloadAttributesV2, error) {
 	if a == nil {
 		return nil, errNilPayloadAttribute
 	}
-	if a.version != version.Capella {
-		return nil, consensus_types.ErrNotSupported("PayloadAttributePbV2", a.version)
+	if a.version < version.Capella {
```

Comment:
why is the conditional changed

Reply:
This is indeed a bug. PbV2 should be the same for deneb but not anymore

Reply:
ill take this

Reply:
#12812